aries-site/src/examples/templates/dashboards/content/UpdatesAvaliable.js

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+import { UpdatesFeed, UpdateNotificationsList } from '../components';
+
+const MOCK_DATA = require('../../../../data/mockData/firmware-bundles.json');
+
+export const UpdatesAvaliable = () => {
+  const data = MOCK_DATA['firmware-bundles'].items;
+  const [updates, setUpdates] = useState([]);
+
+  useEffect(() => {
+    if (data) {
+      const updateArray = [];
+      const sortedDates = data.sort((a, b) => {
+        const c = new Date(a.releaseDate);
+        const d = new Date(b.releaseDate);
+        return d - c;
+      });
```

Comment:
I think there is something off in the sort logic here. 

Currently "2020.03.3(10 Dec 2020)" is shown as the hotfix, however "2020.09.1(09 Dec 2020)" is more recent.

Currently "2020.09.1(22 Apr 2021)" is shown as the base, however "2021.04.0(10 Apr 2021)" is more recent.

Reply:
uh yes thank you I fixed this should be fine now